src/Analyzers/CSharp/Analyzers/UseExpressionBody/Helpers/UseExpressionBodyForMethodsHelper.cs

```diff
@@ -57,7 +55,7 @@ protected override MethodDeclarationSyntax WithBody(MethodDeclarationSyntax decl
             {
                 // if it's 'async TaskLike' (where TaskLike is non-generic) we do *not* want to
                 // create a return statement.  This is just the 'async' version of a 'void' method.
-                var method = semanticModel.GetDeclaredSymbol(declaration);
+                var method = semanticModel.GetDeclaredSymbol(declaration)!;
```

Comment:
is this nullable suppression necessary? 

can't we just do 

```csharp
return method?.ReturnType is INamedTypeSymbol namedType && namedType.Arity != 0;
```

below?

Reply:
@jmarolf Sorry I missed this PR.

I think we should never be getting a null from `GetDeclaredSymbol`, but I'll update to avoid the suppression.